recipes/recipe-Ayurvedic-Kitchari.html

```diff
@@ -0,0 +1,171 @@
+                </ul>
+            </nav>
+
+            
+            <div class="featured_recipe" id="recipe_page">
+                <div class="title_text">
+                    <a href="../recipes.html#recipes"><i class="fas fa-chevron-left"></i></a>
+                    <h2>Ayurvedic Kitchari</h2>
+                </div>
+            </div>
+                
+            <div class="recipe_image_text_section">
+                <img class="photo left" src="../images/recipes/Ayurvedic-Kitchari.jpeg" alt="Recipe"/>
+<!--                <iframe class="recipe-video left" src="https://www.youtube.com/embed/hSJuW2qgmfk" title="YouTube video player" frameborder="0" allow="accelerometer; autoplay; clipboard-write; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>-->
+                <div class="recipe_body right">
+                    <div>
+                        <h3>Recipe submitted by Stanley</h3>
+                        <p class="recipe_info">
+                            <i class="fas fa-seedling"></i>Vegan&ensp;   
+                            <i class="fas fa-carrot"></i>Vegetarian&ensp;
+                            <i class="fas fa-ban"></i>Gluten-Free&ensp;
+                            <i class="fas fa-circle"></i>Easy&ensp;
+    <!--                        <i class="fas fa-square"></i>Moderate&ensp;-->
+    <!--                        <i class="fas fa-star"></i>Difficult&ensp;-->
+                            <br>
+                            <i class="fas fa-clock"></i>55 mins&ensp;
+                            <i class="fas fa-user"></i>3 Servings
+                        </p>
+<!--
+                        <p>Description: 
+                            <a href="https://www.youtube.com/watch?v=tUi4_ektVn0" target="_blank"> Video 1</a>
+                        </p>
+-->
+                    </div>
+                </div>
+            </div>
+            
+            
+            <div class="recipe_image_text_section" id="recipe_image_text_section_body">
+                
+                <div class="recipe_text left">
+                    <h3>Ingredients</h3>
+                    <ul>
+                        <li>½ Cup Moong Dal (split mung beans) </li>
+                        <li>½ Cup Basmati Rice, rinsed</li>
+                        <li>1 Tbsp. of Ghee or Coconut Oil (vegan) </li>
+                        <li>1 tsp Ground Cumin Seeds</li>
+                        <li>1 tsp Mustard Seeds</li>
+                        <li>2 tsp Ground Turmeric</li>
+                        <li>1 tsp Ground Coriander</li>
+                        <li>1 tsp Ground Fennel Seeds</li>
+                        <li>A large 1 inch piece of ginger</li>
+                        <li>1½ Cup mix of veggies</li>
+                        <li>5 cups of water</li>
+                        <li>1 tsp Sea Salt (more for more taste) </li>
+                        <li>Cilantro for garnish</li>
+                        <li>Lentil</li>
```

Comment:
```suggestion
                        <li>½ Cup Lentils</li>
```
Would also move this to the 3rd item since it is related to the Moong Dal and Basmati Rice